From 1eb9bea38c320b2b588635cffceaaa2a8d434780 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 25 Jan 2023 22:09:26 -0800
Subject: [PATCH] include missing <cstdint>

gcc 13 moved some includes around and as a result <cstdint> is no longer transitively
included [1]. Explicitly include it for uint{32,64}_t.

[1] https://gcc.gnu.org/gcc-13/porting_to.html#header-dep-changes

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Upstream: https://github.com/graeme-hill/crossguid/pull/67

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 include/crossguid/guid.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/crossguid/guid.hpp b/include/crossguid/guid.hpp
index 61e0f17..70966f2 100644
--- a/include/crossguid/guid.hpp
+++ b/include/crossguid/guid.hpp
@@ -29,6 +29,7 @@ THE SOFTWARE.
 #include <jni.h>
 #endif
 
+#include <cstdint>
 #include <functional>
 #include <iostream>
 #include <array>
